core/services/pipeline/task.bridge.go

```diff
@@ -89,12 +86,11 @@ func (t BridgeTask) getBridgeURLFromName() (url.URL, error) {
 	return bridgeURL, nil
 }
 
-func withIDAndMeta(request HttpRequestData, runID int64, meta HttpRequestData) HttpRequestData {
+func withMeta(request HttpRequestData, meta HttpRequestData) HttpRequestData {
 	output := make(HttpRequestData)
 	for k, v := range request {
 		output[k] = v
 	}
-	output["id"] = fmt.Sprintf("%d", runID)
```

Comment:
runID doesn't exist for OCR jobs because we don't use UUIDs so can't pre-allocate. Also unused by the adapters so leaving it out